src/main/js/README.md

```diff
@@ -0,0 +1,14 @@
+# Setting up Speech to Text Demo
+
+```sh
```

Comment:
Actually this is not just exclusively for OS X. I'm using, https://github.com/maxogden/menubar, can run on Mac OS X, Windows, and some desktop environments on Linux (e.g. tested on xfce) at least.